Chatkit/View Models/TypingUsersViewModel.swift

```diff
@@ -0,0 +1,82 @@
+import Foundation
+
+/// A view model which provides a text representations of typing users.
+public class TypingUsersViewModel {
+    
+    // MARK: - Properties
+    
+    /// The text representations of typing users.
+    public private(set) var value: String?
+    
+    /// The typing users provider used as the source of data.
+    public let provider: TypingUsersProvider
```

Comment:
Surely this is private? What's the benefit in exposing it?

Reply:
Well it has been injected into the class, it is being retained by the class, so it make sense for this to be exposed as `public`. It is consistent with other SDKs. It also improves testability.

Reply:
After the discussion the property is now marked as `private`.